tests/helix/send-to-helix-inner.proj

```diff
@@ -10,6 +10,8 @@
     <DotNetCliPackageType>sdk</DotNetCliPackageType>
     <NeedsCreateDotNetDevScripts Condition="'$(OS)' != 'Windows_NT'">true</NeedsCreateDotNetDevScripts>
     <HasDocker Condition="'$(OS)' != 'Windows_NT'">true</HasDocker>
+    <!-- Disable playwright tests on helix/linux. Issue: https://github.com/dotnet/aspire/issues/4623 -->
```

Comment:
Is this playright stuff related to this change?

Reply:
hmm this can be extracted as a separate change. It is essentially making sure that we don't bundle, and set up things for running playwright tests when it is disabled for that platform - linux, in this case.

Reply:
Reverted.